odoo/addons/base/wizard/base_import_language.py

```diff
@@ -19,7 +19,7 @@ class BaseLanguageImport(models.TransientModel):
     _description = "Language Import"
 
     name = fields.Char('Language Name', required=True)
-    code = fields.Char('ISO Code', size=6, required=True,
+    code = fields.Char('ISO Code', size=8, required=True,
```

Comment:
This is not a valid change for a stable version, since this modifies the database schema!
That being said, this `size` parameter is actually inconsistent because there is no such size on field `res.lang.code`.  Ideally there should be no size at all.

Reply:
Hi @rco-odoo, thank you for the feedback. So basically I should change this request to a non-stable version, like which version? And I should remove the size.

Reply:
Hi, only in the master branch.